roles/nova-common/tasks/ceph_integration.yml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
The task for nova-configurations has dependency on the inclusion of ```ceph-integration.yml``` to fetch  value for the param ```cinder_uuid_file```. And since the nova-configurations task for nova-compute node are written in nova-common, we have included ceph-integration before the nova config task.